plugins/reach/lib/model/data/VendorLiveTranslationCatalogItem.php

```diff
@@ -0,0 +1,12 @@
+<?php
+/**
+ * @package plugins.reach
+ * @subpackage model
+ */
+class VendorLiveTranslationCatalogItem extends VendorLiveCaptionCatalogItem
```

Comment:
need to create a class that both love caption and live translation can inherit from it